src/UNL/Officefinder/Auth.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+class UNL_Officefinder_Auth
```

Comment:
A general comment regarding methods in this class. Is there a reason that method comments that describe the purpose, parameters and return values were not included?